google_bigquery_writer/app.py

```diff
@@ -52,11 +52,15 @@ def run(self):
         if len(parameters) == 0:
             message = 'Configuration is empty.'
             raise UserException(message)
-        if parameters.get('dataset') is None or parameters.get('dataset') == '':
-            message = 'Google BigQuery dataset not specified in the configuration.'
+        if parameters.get('dataset') is None or \
+                parameters.get('dataset') == '':
+            message = \
+                'Google BigQuery dataset not specified in the configuration.'
```

Comment:
Tady bychom mozna mohli nastavil flake, aby dovolil mit radek dlouhy 120 znaku. Tech 80 je zbytecna buzerace. Musis potom lamat kazdou message na vice radku. Nechtel bych tem kontrolam delat ustupky, ale tady si myslim, ze to tak striktni byt nemusi.

http://flake8.pycqa.org/en/latest/user/options.html#cmdoption-flake8-max-line-length

Reply:
Jo, dobrej nápad, mě těch 80 znaků děsně 💩 :-)